src/main/java/io/confluent/idesidecar/restapi/resources/RestProxyResource.java

```diff
@@ -43,16 +57,65 @@ public void schemaRegistryClusterProxy(RoutingContext routingContext) {
   }
 
   private void handleClusterProxy(RoutingContext routingContext, ClusterProxyContext proxyContext) {
-    clusterProxyProcessor.process(proxyContext)
+    process(routingContext, clusterProxyProcessor, proxyContext);
+  }
+
+  private void handleRBACProxy(RoutingContext routingContext, ProxyContext proxyContext) {
+    process(routingContext, rbacProxyProcessor, proxyContext);
+  }
+
+  @Route(
+      path = RBAC_RESOURCE_PATH,
+      methods = Route.HttpMethod.PUT,
+      produces = MediaType.APPLICATION_JSON,
+      consumes = MediaType.APPLICATION_JSON
+  )
+  @Blocking
+  public void RBACProxyRoute(RoutingContext routingContext) {
+    handleRBACProxy(routingContext, createRBACProxyContext(routingContext));
+  }
```

Comment:
It would be nice to add the request/response schema here so that it's part of the generated openapi spec. The way to do that would be to create Java `record` types that exactly represent the schema, and then specifying that here. 

cc @shouples